petcare-sample/android-mobile-application/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
S. all we add a newline here?  

Check other places as well.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/443/commits/a0de749a7604845679f1995fc79c5815c95f7a5d